server/src/tests/routes/user.test.ts

```diff
@@ -5,6 +5,8 @@ import { createCaller } from '../../routes/trpcRouter';
 
 beforeEach(async () => {
   process.env.NODE_ENV = 'test';
+  process.env.SEND_GRID_API_KEY="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"
```

Comment:
don't leak secret keys, sendgrid blocks it everytime. that's why the keys are always unauthorized :(

Reply:
removed it.